assets/section-main-product.css

```diff
@@ -648,6 +681,40 @@ a.product__text {
     right: 5rem;
     top: 2.2rem;
   }
+  .image_label {
```

Comment:
The styles for .image_label and #image_type seem to be duplicated in this file. Another copy is above.

Reply:
Still duplicate, but did some cleaning. Another ones are for smaller media screen size (Jordans proposal).

Reply:
oooh. My mistake. I didn't see the context enough to realize that it was in a media query.
But I also guess it didn't require all the parameters to be redefined :)